### PR TITLE
128 building unique job pages

### LINKED ISSUE #128: Building a page for each job so it has it's own unique URL + SEO
**Features being implemented**
 Build a page for each job so it has its own unique URL + SEO
 Allow for a brief description of the job (if we build individual job pages)
 Allow for a brief company or job summary

**Describing the solution**
Each job would have its own page under https://denverdevs.org/jobs/[id]. The job page would showcase information already found in the preview card plus a company summary and job description fields. The jobs page is meant to give the user more details about the job and even possibly the company who posted the job. Each page will have a unique title and description in the header for SEO purposes.

**Describe alternatives you've considered**
I am open to suggestions about any alternatives to my implementation :)

**Additional context**
Style-wise, I am going off of features already present on the site. I am open to any design suggestions as well.

### PR DESCRIPTION
**Pull request by ChaseDudas**
### Why

Closes #128 

@danhannigan I had some writer's block while making the SEO description for the head tag. Do you think the copy for that statement could be improved?

### What is changing

- Builds a page for each job so it has its own unique URL ( [https://denverdevs.org/jobs/[id]](https://denverdevs.org/jobs/%5Bid%5D) )

- Adds a simple SEO page description, title, and other tags using data we already have

- Allows a job poster to give a brief description of the job ( Twitter style. Text-area input with char limit. Give us your elevator pitch! )

### CODE REVIEW

**Review by danhannigan — CHANGES_REQUESTED** [review]
Thanks for tackling a lot of this! Popped open a few feedback points to clear up before we go on and merge it. Took a look at the deploy preview and that all looks fine to me from a visual standpoint, though. SEO Copy looks good enough to me!

File: pages/jobs/[id]/index.jsx
Comment (ChaseDudas) [reply pages/jobs/[id]/index.jsx:20]: It can! Learnt a new thing today. Thank you for the suggestion @danhannigan

File: pages/jobs/post-job.jsx
Comment (ChaseDudas) [reply pages/jobs/post-job.jsx]: I don't mind at all 👍

File: pages/jobs/[id]/index.jsx
Comment (danhannigan) [reply pages/jobs/[id]/index.jsx]: Rad! Yeah that's a big appeal of Next is their data-fetching, and basically, this will statically render it on the Netlify site so we don't have to worry about loading and stuff! Super fast — I need to swap the data fetch on the main job index page to do this, too. The only downer is that it requires a rebuild on the server to have the latest content but the jobs already need manual approval, so, I can just kick off a build while I'm at it too.

**Review by danhannigan — APPROVED** [review]
Looks great! Thanks for all the hard work on this, I'm pretty excited for it

**Pull request merged**
